Basic Create Edit Del

MadeByDouglas opened the pull request:
basic implementation is all functional

MadeByDouglas committed:
Basic Create Edit Del

basic implementation is all functional

falkobuttler commented on com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m:
```diff
@@ -24,13 +29,45 @@ - (void)setDetailItem:(id)newDetailItem {
         [self configureView];
     }
 }
+- (IBAction)saveNote:(id)sender {
```
If you want to save on closing the view, you could move this to viewDidDisappear. This functionality makes only sense for existing notes, though.

MadeByDouglas replied:
ah ok, yes i will move the save content into its own method which the button or viewDidDisappear will call. I am modifying the save method to account for a "blank" new note. Also, I guess this sorta makes the save button redundant, but perhaps is reassuring to the user.

MadeByDouglas committed:
Create Edit Delete Completed

textView fixed (autoAdjustsScrollViewInserts)
save properly saves or updates items, and ignores blank or no-change
items
Save button correctly dismisses DetailVC (removed redundant VC and
added new segue)
